### ISSUE TITLE
Dashboard's JSON model inconsistency

### ISSUE THREAD

**Issue by rivalver** [VictoriaMetrics/grafana-datasource#23]
Hello!

JSON for the freshly created dashboard, which has only 1 VictoriaMetrics data source has some inconsistencies.
Datasource is mentioned in JSON data twice, but with different types: 
1. `$.panels[0].datasource.type = "victoriametrics-datasource"`
2. `$.panels[0].targets.datasource.type = "prometheus"`

This could potentially make any automations harder.

Dashboard example

```
{
  "annotations": {
    "list": [
        ///
  },
  "editable": true,
  "fiscalYearStartMonth": 0,
  "graphTooltip": 0,
  "links": [],
  "liveNow": false,
  "panels": [
    {
      "datasource": {
        "type": "victoriametrics-datasource",
        "uid": "000032369"
      },
      "fieldConfig": {
        "defaults": {
          "color": {
            "mode": "palette-classic"
          },
          "custom": {
            "axisCenteredZero": false,
            "axisColorMode": "text",
            "axisLabel": "",
            "axisPlacement": "auto",
            "barAlignment": 0,
            "drawStyle": "line",
            "fillOpacity": 0,
            "gradientMode": "none",
            "hideFrom": {
              "legend": false,
              "tooltip": false,
              "viz": false
            },
            "lineInterpolation": "linear",
            "lineWidth": 1,
            "pointSize": 5,
            "scaleDistribution": {
              "type": "linear"
            },
            "showPoints": "auto",
            "spanNulls": false,
            "stacking": {
              "group": "A",
              "mode": "none"
            },
            "thresholdsStyle": {
              "mode": "off"
            }
          },
          "mappings": [],
          "thresholds": {
            "mode": "absolute",
            "steps": [
              {
                "color": "green",
                "value": null
              },
              {
                "color": "red",
                "value": 80
              }
            ]
          }
        },
        "overrides": []
      },
      "gridPos": {
        "h": 11,
        "w": 24,
        "x": 0,
        "y": 0
      },
      "id": 2,
      "options": {
        "legend": {
          "calcs": [],
          "displayMode": "list",
          "placement": "bottom",
          "showLegend": true
        },
        "tooltip": {
          "mode": "single",
          "sort": "none"
        }
      },
      "targets": [
        {
          "datasource": {
            "type": "prometheus",
            "uid": "000032369"
          },
          "editorMode": "code",
          "expr": "emsv2_stats_open_alerts_count{}",
          "legendFormat": "__auto",
          "range": true,
          "refId": "A"
        }
      ],
      "title": "Open Alerts Count",
      "type": "timeseries"
    }
  ],
  "schemaVersion": 37,
  "style": "dark",
  "tags": [],
  "templating": {
    "list": []
  },
  "time": {
    "from": "now-6h",
    "to": "now"
  },
  "timepicker": {},
  "timezone": "",
  "title": "New dashboard",
  "version": 0,
  "weekStart": ""
}
```

Grafana version: 9.2.4

**Comment by Loori-R** [comment]
Hi @rivalver. This should be addressed in commit 72be3715bb8340817834784e0a33e4d2c4d9ae30, which will be included in the next release.

**Comment by Loori-R** [comment]
The fix has been included in [victoriametrics-datasource v0.1.2](https://github.com/VictoriaMetrics/grafana-datasource/releases).
